icx/README.md

```diff
@@ -38,3 +38,31 @@ If you have a candid file, you can use it to validate arguments. Pass it in with
 ```shell script
 icx query 75hes-oqbaa-aaaaa-aaaaa-aaaaa-aaaaa-aaaaa-q greet --candid=~/path/greet.did '("World")' 
 ```
+
+#### Sign and Send Separation
+Pass `--serialize` when use `icx update` and `icx query` will serialize the signed canister call message as json.
+The output will print to stdout. When use `icx update`, a corresponding request_status message is also generated and printed to stderr.
+
+In the default IC project generated with `dfx new` and the local emulator has started with `dfx start --background`.
+
+##### Sign
+```shell script
+icx --fetch-root-key update --serialize rwlgt-iiaaa-aaaaa-aaaaa-cai greet '("everyone")' 1> update.json 2>request_status.json
+```
+> `rwlgt-iiaaa-aaaaa-aaaaa-cai` is the ID of hello canister in the default project.
+
+##### Send
+```shell script
+cat update.json | icx send
+...
+RequestID: 0x1234....
+```
+
+##### Request status
+```shell script
+cat request_status.json | icx --fetch-root-key send
```

Comment:
If the default is hard-coded key, this `--fetch-root-key` doesn't seem necessary.